doc/en/developer/source/release/guide/index.rst

```diff
@@ -155,8 +155,6 @@ When creating the first release candidate of a series, there are some extra step
 * Switch to the new branch and update the documentation links, replacing ``docs.geoserver.org/latest`` with ``docs.geoserver.org/2.12.x`` (for example):
    
   * ``README.md``
-  * ``doc/en/developer/source/conf.py``
```

Comment:
I think the ant build/rename.xml has already updated these 2 conf.py files, hasn't it @jodygarnett ?

Does the README.md still need to be changed - probably yes.

Reply:
I hope the rename.xml does what is needed. But I still wish to document the manual process (if needed).